main.py

```diff
@@ -24,17 +25,114 @@ def setup(self):
         self._flow_mods_sent = OrderedDict()
         self._flow_mods_sent_max_size = FLOWS_DICT_MAX_SIZE
 
+        # Storehouse client to save and restore flow data:
+        self.storehouse = StoreHouse(self.controller)
+
+        # Format of stored flow data:
+        # {'flow_persistence': {'dpid_str': {'flow_list': [
+        #                                     {'command': '<add|delete>',
+        #                                      'flow': {flow_dict}}]}}}
+        self.stored_flows = {}
+        self.resent_flows = set()
+
     def execute(self):
         """Run once on NApp 'start' or in a loop.
 
         The execute method is called by the run method of KytosNApp class.
         Users shouldn't call this method directly.
         """
+        self._load_flows()
 
     def shutdown(self):
         """Shutdown routine of the NApp."""
         log.debug("flow-manager stopping")
 
+    @listen_to('kytos/topology.port.created')
```

Comment:
Yes, the method has been updated(https://github.com/kytos/flow_manager/pull/90/commits/cca0f32f68e8b7fc88b03b6816a309239db3ef55) to listen to `kytos / of_core.handshake.completed` instead of` kytos / topology.port.created`